src/ui/public/utils/decode_geo_hash.js

```diff
@@ -1,18 +1,23 @@
 /*
- * Decodes geohash to object containing
- * top-left and bottom-right corners of
- * rectangle and center point.
- *
  * geohash.js
  * Geohash library for Javascript
  * (c) 2008 David Troy
  * Distributed under the MIT License
+ */
+
+/*
+ * @notice
+ * This product bundles geohash.js which is available under a
+ * "MIT" license.  For details, see src/ui/public/utils/decode_geo_hash.js.
```

Comment:
__typo:__ redundant space in front of `For details...`

Reply:
Yeah, it was in the source so I left if to keep the diff minimal, but I can remove it.

Reply:
Ah, then don't bother